state/dbStateManager.go

```diff
@@ -1336,15 +1349,6 @@ func (list *DBStateList) SaveDBStateToDB(d *DBState) (progress bool) {
 		panic(err.Error())
 	}
 
-	// Set the Block Replay flag for all these transactions we are saving to the database.
-	for _, fct := range d.FactoidBlock.GetTransactions() {
-		list.State.FReplay.IsTSValid_(
```

Comment:
This code was moved around to catch when loading from blocks and downloading from messages